drivers/gles2/rasterizer_canvas_gles2.cpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
Not sure what is going on here, but it should be fixed.

Reply:
Installed a hexeditor, turns out my edited file had a UTF8 BOM (3 hex code) at the start of the file, I'm presuming Geany or another editor must have added this at some point! :grimacing:  Have changed QT creator to delete these if found in a source file. We might be able to add this to the static clang checks, I'll ask Akien / Calinou.